JotunnLib/Managers/GUIManager.cs

```diff
@@ -1360,6 +1360,10 @@ public void ApplyToogleStyle(Toggle toggle)
         /// <param name="fontSize">Optional font size, defaults to 16</param>
         public void ApplyDropdownStyle(Dropdown dropdown, int fontSize = 16)
         {
+            // Move the Dropdown and ScrollRect components to the UILayer
+            dropdown.gameObject.layer = UILayer;
+            dropdown.transform.Find("Template").gameObject.layer = UILayer;
```

Comment:
The template child shouldn't be hardcoded since it may be named differently or be active under a second child. `dropdown.template` should work better and I think a null check is also appropriate since any dropdown instance may be passed here.

Reply:
Ah, I didn't know it had a template property. Sorry, not very familiar with Unity's UI elements. Good call on the null check, I made assumptions there.

I've pushed to the branch on my fork - should it automatically appear here? I've never done a PR before, sorry.

Reply:
Yes looks good now. Yeah the changes are included here now, a PR always points to a branch :D

Reply:
Splendid, sorry to be a pain! Is there anything else I need to do?

Reply:
It's all good, thanks for your contribution. We all been at our first PR and this is what reviews are for :)
I'm going though a few sanity checks now, compile the changes and do a few tests to not miss something obvious and will merge it afterwards.